### ISSUE TITLE
[Progression][Classic] Engineering Schematics

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#6560]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

Generic

### Current Behaviour

Engineering skills which should be learned through schematics were available via trainers
https://wowgaming.altervista.org/aowow/?spell=19795
https://wowgaming.altervista.org/aowow/?spell=19791

### Expected Blizzlike Behaviour

these crafting skills should not be available from trainers but require the schematic
https://www.wowhead.com/classic/item=16042/schematic-thorium-widget
https://www.wowhead.com/classic/item=16047/schematic-thorium-tube

### Source

.

### Steps to reproduce the problem

.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/c863a8cfb59ed17692f9da2956b85a53ecc09ed7

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Cooties-cc** [comment]
Seems misleading, as links were given in **_vanilla classic_**. If you're trying to add to the prog branch, you need to do at least **_TBC classic data._** That of which lists these as being learned from a trainer, or by pattern. 
https://www.wowhead.com/tbc/item=15994/thorium-widget#taught-by-npc
https://www.wowhead.com/tbc/item=16000/thorium-tube#taught-by-npc

Further digging shows earliest comment about the swap (for widget) at patch 2.0.12.
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/501cd368-c2b7-410d-99e4-61a562c0748d)
Wowpedia, if taken as source correct, lists the change around patch 2.0.8 for the widget, but not the tube
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/f953b5e8-9009-4926-989a-0a18845d0df0)
Another later TBC comment at patch 2.1.3 list further patterns that were converted to be learned from a trainer as well. 
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/2dcfc3b6-7613-4ab1-9608-45cfd83e321e)

I think like class tomes, later version of the game allowed them to be trained from either source.

**Comment by heyitsbench** [comment]
> If you're trying to add to the prog branch, you need to do at least TBC classic data.

Not necessarily, it seems the staff are open to retroactively adding more thorough progression changes to vanilla content as well, as can be seen from [this issue being triaged](https://github.com/chromiecraft/chromiecraft/issues/6505).

**Comment by Cooties-cc** [comment]
> Not necessarily, it seems the staff are open to retroactively adding more thorough progression changes to vanilla content as well

This doesn't make sense to apply to vanilla things that were changed in current "PTE phase". We're done with vanilla "progression", thus the progression phase should be matching current content (between 2.0.3ish and 2.1 atm). The thorium tube and widget were changed at the same level of progression we're playing at (2.0.8ish, but pre 2.1). Looking at the triaged items, they don't follow the same progression release timeline, so they aren't quite comparable. 

**Mooncloth** : 3 days 22 hours + moonwell was required _until patch 2.4.3_. This is appropriate to ask for currently. 
**Transmute Arcanite** : 2 day CD until _patch 2.1_, CD removed at _patch 2.4_. This is appropriate to ask for currently. 
**Salt Shaker** : 3 day CD, wasn't changed till seemingly _patch 3.1_ when the CD was removed. This is appropriate to ask for currently. 
**Alchemy Labs** : They were required to create flasks until _patch 3.0.8_. However there were 4 in TBC total, 3 of which we can access atm which were the one in Scholo, and one in BWL, as well as the one Shattrath City in lower city northwest of the aldor bank. The 4th one being at the Isle of Quel'Danas when the dailies open later. This is appropriate to ask for currently. 

Those are still affected by the prog branch at current PTE "phase", and should be until we reach that level of content to alter them. The engineer items in this ticket however do not meet the same threshold from what I can see.

**Comment by balleny** [comment]
The intention of the ticket was to create awareness that there were things missed in the past progression modules that may or may not should be included if there is the intention to provide a rather _complete and correct_ progression.

https://github.com/azerothcore/mod-progression-system
> DISCLAIMER This module is used by Chromiecraft and the progression order here presented will oftentimes reflect Chromiecraft's progression. If you would like a different progression you are free to change the order of the files (file locations) and scripts as you would like.


the module is CC specific and hence there may be a decision to not make adjustments to past progressions (e.g. because they wont be tested either). But, my ticket here is so that at least the ticket can be moved to the repository and listed as an issue. That way, if anyone decides to utilize the progression module in the sense of open source, would also see issues and deviations from original vanilla progression.

**Comment by balleny** [comment]
i did not put any effort into listing for how long certain classic things should be active or at which point in time from vanilla onwards they got changed to what we see right now being wotlk default. but for progression purpose, it does make sense to have those implemented - or at least noted so that others can fork and aim for a more original profession with less wotlk changes still active

**Comment by Cooties-cc** [comment]
Yeah that seems fine then. It sorta came off looking like a retroactive request to change something to the vanilla timeline, which we're already doing TBC prog changes, on a wotlk client with those changes not fixed or unable to change. If its more just for clarity and visibility about the prog branch itself, so others can see a change needing a flag of sorts if they were to have it on another AC server starting in vanilla, like we were doing, that makes sense to have them be aware and flag the change.

**Comment by FrancescoBorzi** [comment]
what is the status of this?
